recipes/scikit-cycling/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "scikit-cycling" %}
+{% set version = "0.1.3" %}
+{% set sha256 = "3acaed9ccfdfa34f43e75342bd86c18bb3ad4b940a24eedb78f674364fba1076" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/scikit-cycling/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script:
+    - export CC=clang  # [osx]
+    - export CXX=clang++  # [osx]
```

Comment:
This is unusual. Have you tried compiling without the clang options?  

It is worth checking the numpy, skit-learn feedstock recipes to see what they do. If they do the same thing then just reply back this is the case.

Reply:
I don't know for numpy. Scikit-learn rely on joblib which used multiprocessing up to know (they move with loky so that could change thing). Due to this issue, they cannot use openmp so they don't care in macos for enabling openmp.

In my case I need to enable openmp. However, clang in osx does not come with openmp support. I used the same recipe as xgboost to allow openmp support in osx by using clangdev from conda.

Reply:
So the `toolchain` does this for you actually. That said, there are no problems caused by doing this here. Would just recommend dropping it in the feedstock.

ref: https://github.com/conda-forge/toolchain-feedstock/blob/57df6a53a7eca4bc6a7b70473e481961a478be93/recipe/activate.sh#L6-L7

Reply:
Great.